ServiceApp/server/src/utils/credentialHelper.ts

```diff
@@ -29,7 +31,78 @@ export interface IUser {
     address: string;
 }
 
-export function createNewUser(name: string = '', role: string = '', location: string = ''): Promise <IUser> {
+export function createNewUserC2(name: string = '', role: string = '', location: string = ''): Promise<IUser> {
+    return new Promise<IUser>(async (resolve, reject) => {
+        const clientConfig = {
+            network: "main",
+            node: "https://chrysalis-nodes.iota.org:443",
```

Comment:
Please move this into a config

Reply:
fixed in 3c6978c6f75aa8ed7718b8a8b6f9e08a561585a2